executor/insert_common.go

```diff
@@ -904,6 +904,9 @@ func (e *InsertValues) allocAutoRandomID(fieldType *types.FieldType) (int64, err
 }
 
 func (e *InsertValues) rebaseAutoRandomID(recordID int64, fieldType *types.FieldType) error {
+	if recordID < 0 {
+		return nil
+	}
```

Comment:
rebase in allocator seems do the same thing (rebase < cur base return nil), so here for returning quickly?

Reply:
`auto_random` only rebase the incremental bits(lower bits). It is always positive, we prevent the case like `recordID < 0 < currentID < lower_bits(recordID)`.

Reply:
got